pkg/controllers/management/drivers/dynamic_driver.go

```diff
@@ -80,9 +80,13 @@ func (d *BaseDriver) Executable() error {
 	if err != nil {
 		return fmt.Errorf("Driver %s not found", binaryPath)
 	}
-	err = exec.Command(binaryPath).Start()
+	cmd := exec.Command(binaryPath)
+	err = cmd.Start()
 	if err != nil {
 		return errors.Wrapf(err, "Driver binary %s couldn't execute", binaryPath)
 	}
+
+	// We don't care about the exit code, just want to make sure we can execute the binary
+	_ = cmd.Wait()
```

Comment:
Should we be concerned that we're blocked here until `cmd` finishes?

Reply:
The command exits (with code 1) immediately because we aren't calling it with any arguments. I can put this in a go-routine if you'd like, but I don't think that is necessary.

Reply:
That's fine then - thank you for clarifying how it's used here.